docs/source/blocks/blocks-core/index.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I would like to rename this new directory to avoid repetition, as well as the references to it.

`docs/source/blocks/core/`

Reply:
Done